code/modules/black_market/illegalradio.dm

```diff
@@ -0,0 +1,370 @@
+#define MAIN 1
+#define SELLING 2
+#define DELIVERY 3
+
+var/list/global_illegal_radios = list()
+
+/proc/buzz_black_market() //Updates and buzzes all illegalradios so the screen is current.
+	for(var/obj/item/device/illegalradio/radio in global_illegal_radios)
+		if(radio.notifications && usr && radio in usr.contents)
+			radio.interact(usr)
+			to_chat(usr,"<span class='info'>You feel \the [radio] buzz.</span>")
+
+/obj/item/device/illegalradio/nanotrasen
```

Comment:
you actually don't need this empty definition at all, it's fine to just have procs for a subtype